burner-ui/src/Modals/Send/Send.tsx

```diff
@@ -372,14 +373,14 @@ class SendModal extends Component<SendPageProps, SendPageState, BurnerContext> {
                       >
                         Max
                       </MaxButton>
-                      {/* <AssetSelector
+                      <AssetSelector
                         selected={asset}
                         assets={assets}
                         // onChange={() => this.setState({ asset: newAsset })}
```

Comment:
This should be uncommented, right?

Reply:
I don't see that on my version of the file...

Reply:
This is how I see it: 

```  <AssetSelector
                        selected={asset}
                        assets={assets}
                        // onChange={() => this.setState({ asset: newAsset })}
                        disabled={sending}
                      />```

No errant `{/*`

Reply:
Was talking about the onChange line